pkg/apis/kops/validation/validation.go

```diff
@@ -266,6 +266,13 @@ func validateKubeAPIServer(v *kops.KubeAPIServerConfig, fldPath *field.Path) fie
 		}
 	}
 
+	if v.AuthorizationMode != nil && strings.Contains(*v.AuthorizationMode, "Webhook") {
+		if v.AuthorizationWebhookConfigFile == nil {
+			flds := [2]*string{v.AuthorizationMode, v.AuthorizationWebhookConfigFile}
```

Comment:
That's an unusual syntax - I would have expected you could just use `[]*string{ v.AuthorizationMode, v.AuthorizationWebhookConfigFile} `

Reply:
I literally wrote my first lines of Go last week, so any unusual syntax is more due to incompetence rather than any desire to be different ;) In this case though I think I saw this syntax used on line 257 above and figured there's probably a point in doing it like that.